ws-nextjs-app/pages/[service]/live/[id]/[[...variant]].page.tsx

```diff
@@ -147,7 +147,7 @@ export const getServerSideProps: GetServerSideProps = async context => {
     page,
     service,
     variant,
-    rendererEnv,
+    rendererEnv: rendererEnv || 'test', // TODO: remove hardcoding
```

Comment:
Whatever the live page code is it will always default the live pages to fetch from test fabl, but we want to control this with the rendere_env instead. No hard coding allowed 👿